x-pack/test/api_integration/apis/management/cross_cluster_replication/follower_indices.js

```diff
@@ -74,17 +74,17 @@ export default function ({ getService }) {
 
     describe('get()', () => {
       it('should return a 404 when the follower index does not exist', async () => {
-        const name = getRandomString();
```

Comment:
@sebelga I removed this helper in favor of explicit hard-coded values for a couple reasons:
* When debugging a test gone wrong (for example, clean up code that deletes these primitives), explicit hard-coded values can provide debugging information
* I generally try to simplify systems by removing any code that isn't necessary, and it didn't seem necessary to me to randomize these values

Do I have your blessing to make this change? :)

Reply:
Agreed on the reasoning. You do have my blessing! 👍  :)

Reply:
Thank you, Seb!